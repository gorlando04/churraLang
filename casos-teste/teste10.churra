amigo "A" contribui 100
amigo "B" contribui 150
amigo "C" contribui 50 vegetariano
item carne "frango" preco_kg 20 quantidade_kg 2
item comida preco_unit 15 quantidade 4
item bebida preco_L 5 quantidade_L 4
rateio igual