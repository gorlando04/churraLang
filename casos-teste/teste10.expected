Total do churrasco: R$120.00
A deve pagar: R$46.67
B deve pagar: R$46.67
C deve pagar: R$26.67